app/routes/logout/route.tsx

```diff
@@ -0,0 +1,17 @@
+import { LoaderFunctionArgs, redirect } from '@remix-run/node';
+import { destroySession, getSession, initializeEcomApiForRequest } from '~/lib/ecom/session';
+
+export async function loader({ request }: LoaderFunctionArgs) {
+    const currentUrl = new URL(request.url);
+    const returnUrlPath = currentUrl.searchParams.get('returnPath') || '/';
```

Comment:
Who's going to set `returnPath`?

Reply:
there is another comment about login and `returnPath`, but for logout I agree that maybe it makes sense to always redirect to home page

Reply:
redirect to home page after logout